setup.py

```diff
@@ -4,9 +4,30 @@
 
 from setuptools import setup
 
+
+if sys.version_info < (3, 6):
```

Comment:
This will execute too if users have `pip < 9.0.1`, which didn't know about `setup(..., python_requires='...')`.